js/app/forms/delivery.js

```diff
@@ -124,8 +124,7 @@ function getTaskType(name, type) {
   return document.querySelector(`#${name}_${type}_type`).value.toUpperCase()
 }
 
-function createDatePickerWidget(name, type) {
-
+function createDatePickerWidget(name, type, userAdmin) {
```

Comment:
When it's an admin, we should show a `DatePicker.RangePicker` instead of a `DateTimePicker`.

Reply:
We're already showing a `DateTimePicker` when store has not a timeslot configured. 
Why should we introduce a new component here instead of use that one?